common/main/hy.xml

```diff
@@ -1616,7 +1616,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 						<dateFormatItem id="Gy">G y թ.</dateFormatItem>
 						<dateFormatItem id="GyMd">dd.MM.y GGGGG</dateFormatItem>
 						<dateFormatItem id="GyMMM">G y թ. MMM</dateFormatItem>
-						<dateFormatItem id="GyMMMd">d MMM, y թ.</dateFormatItem>
+						<dateFormatItem id="GyMMMd">d MMM, y թ. G</dateFormatItem>
```

Comment:
Generic standard formats have this pattern (but there is a lot of variability in this locale among the G patterns in generic and gregorian)